### ISSUE TITLE
Error in the "Usage" sample code

### ISSUE THREAD

**Issue by dandelionred** [reactphp/event-loop#56]
```php
$loop = React\EventLoop\Factory::create();

$server = stream_socket_server('tcp://127.0.0.1:8080');
stream_set_blocking($server, 0);
$loop->addReadStream($server, function ($server) use ($loop) {
	$conn = stream_socket_accept($server);
	$data = "HTTP/1.1 200 OK\r\nContent-Length: 3\r\n\r\nHi\n";
	$loop->addWriteStream($conn, function ($conn) use (&$data, $loop) {
		$written = fwrite($conn, $data);
		if ($written === strlen($data)) {
			fclose($conn);
			$loop->removeStream($conn);
		} else {
// ==========================================
			$data = substr($data, 0, $written);
// Shouldn't it be substr($data, $written) ?
// ==========================================
		}
	});
});

$loop->addPeriodicTimer(5, function () {
	$memory = memory_get_usage() / 1024;
	$formatted = number_format($memory, 3) . 'K';
	echo "Current memory usage: {$formatted}\n";
});

$loop->run();
```

**Comment by clue** [comment]
Thanks for spotting! Do you care to file a PR? :+1:

**Comment by dandelionred** [comment]
PRed

**Comment by clue** [comment]
Closed via #57

Thanks! :+1:
